### ISSUE TITLE
Creating issues to update alt texts on About Us page

### ISSUE THREAD

**Issue by kathrynsilvaconway** [hackforla/website#3361]
### Dependency
 - [x] #3382
 - [x] #3383 
 - [x] #3397
 - [x] #3398 
 - [x] #3402 
 - [x] #3401
 - [x] #3404
 - [x] #3396 
 - [x] #3400 
 - [x] #3399
 - [x] #3403 

### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this epic, we will write issues to change the alt text for the images in the [About Us page](https://www.hackforla.org/about/) which will then be added to dependency items above.

### Details
Designers and PMs identified that some alt texts in the [About Us page](https://www.hackforla.org/about/) should be changed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [x] This issue should only be assigned to a lead.
- [x] Lead would guide other developers on creating issues using the following steps (These steps would be recurring for each issue that would be created and this action item should stay unchecked until all the issues have been created):
    - Check the following [Google spreadsheet tab](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=2047679475) to track the issue creation progress (there are 56 issues in total that need to be made)
    - Choose a row for which issue needs to be created and add your name in the Issue writer column. (If multiple team members creating issues simultaneously, let other team members know which row you would be working on, so there are no duplicate issues).  See column H to know if an issue has been made already
    - Create a new issue with steps to change the alt text for an image following the sample issue below as well as using [How to create issues](https://github.com/hackforla/website/wiki/How-to-create-issues) wiki.
    - Update the issue link in the [Google spreadsheet tab](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=2047679475)
- [x] Add the issue numbers of all the issues that have been created to the dependency section above.
- [x] If all the issues for the spreadsheet tab have been created, add a dependency label and move this issue to ice box column.
- [x] If all the dependency issues are closed, remove the dependency label, move this issue from ice box to In progress column and close this issue.

### Resources/Instructions
[About Us page](https://www.hackforla.org/about/)
[Google spreadsheet tab](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=2047679475) to track issue creation
[How to create issues](https://github.com/hackforla/website/wiki/How-to-create-issues) wiki

### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). In this issue, we will change the alt text for the Radical Inclusivity image on the [About Us page](https://www.hackforla.org/about/).

### Details
Designers and PMs identified that some alt texts in the Events page should be changed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [ ] In the file `_includes/about-page/about-card-accomplishments.html`, add the specified alt code to the following line (the code is intentionally `alt=""` (the tag is supposed to be empty):

```
<img src="/assets/images/about/wins-cup.svg">
```

New Text
```
alt=""
```

OR

### Action Items
- [ ] In the file `_includes/about-page/about-card-sustainability.html`, change the following alt text

From
```
Zero Hunger goal logo
```

To
```
2 Zero Hunger
```

- [x] **For merge team**: Once the pull request associated with this issue is approved and merged, please update and edit epic #3361 by
  - [x] Checking off the dependency for this issue
  - [x] If all dependencies are checked off, please move issue #3361 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is part of epic #3361 
- [About Us page](https://www.hackforla.org/about/)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

#### File and code links you will need to work on this issue
- Directory to find the page once you have it in your IDE: `_includes/about-page/about-card-platform.html`
- Click on the link below to see the source file code:
https://github.com/hackforla/website/blob/a83dd650b664618ff2352b2fb93e8ae4acfb402b/_includes/about-page/about-card-platform.html#L77

#### Information about WCAG
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
